src/avp64/system.cpp

```diff
@@ -108,13 +121,17 @@ system::system(const sc_core::sc_module_name& nm):
     addr_hwrng("addr_hwrng", { HWRNG_LO, HWRNG_HI }),
     addr_spi("addr_spi", { SPI_LO, SPI_HI }),
     addr_gpio("addr_gpio", { GPIO_LO, GPIO_HI }),
+    addr_can("addr_can", { CAN_LO, CAN_HI }),
+    addr_can_messageram("addr_can_messageram", { CAN_MESSAGERAM_LO, CAN_MESSAGERAM_HI }),
```

Comment:
This line is to long (the clang-format file allows 79 characters per column). To keep it in one line, i suggest to rename `CAN_MESSAGERAM_*` to `CAN_MSGRAM_*`.

Reply:
I just noticed that `addr_can_messageram` needs to be shortened to `addr_can_msgram`, too.